### PR TITLE
[docs] Improved pull request template

### PR DESCRIPTION
**Pull request by Panquesito7**
# Pull Request

## Description

- Improved the pull request template.
  - Removed the `:` in headers.
  - Added link to the contributing guidelines in the first Markdown comment.
  - Colorized with black the `NOT` and `DOES` words at the bottom.

## Scope

Did you remember to update the `package.json` version number?

- [ ] Major Version Update: X.\_.\_ (for complete code refactors, or large PRs that adjust most of the codebase)
- [ ] Minor Version Update: \_.X.\_ (for the addition of new commands)
- [ ] Patch Version Update: \_.\_.X (for bug fixes _in the code_.)
- [ ] No Version Update: \_.\_.\_ (no version update for additions to tests, documentation, or anything that isn't end-user facing.)

## Documentation:

For _any_ version updates, please verify if the [documentation page](https://www.beccalyria.com/discord-documentation) needs an update. If it does, please [create an issue there](https://github.com/BeccaLyria/discord-documentation/issues/new?assignees=nhcarrigan&labels=%F0%9F%9A%A6+status%3A+awaiting+triage&template=update.md&title=%5BUPDATE%5D) to ensure it is not forgotten.

- [x] My contribution does **NOT** require a documentation update.
- [ ] My contribution **DOES** require a documentation update.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ Did you remember to update the `package.json` version number?
...
 
-## Documentation:
+## Documentation
 
 For _any_ version updates, please verify if the [documentation page](https://www.beccalyria.com/discord-documentation) needs an update. If it does, please [create an issue there](https://github.com/BeccaLyria/discord-documentation/issues/new?assignees=nhcarrigan&labels=%F0%9F%9A%A6+status%3A+awaiting+triage&template=update.md&title=%5BUPDATE%5D) to ensure it is not forgotten.
 
-- [ ] My contribution does NOT require a documentation update.
-- [ ] My contribution DOES require a documentation update.
+- [ ] My contribution does **NOT** require a documentation update.
+- [ ] My contribution **DOES** require a documentation update.
```
Comment (nhcarrigan) [inline .github/PULL_REQUEST_TEMPLATE.md:29]: I think we should revert this change. Double emphasis (both bold and capital) is excessive IMHO, and we should use only one.

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
One tiny nitpick here.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Panquesito7) [reply .github/PULL_REQUEST_TEMPLATE.md]: Yeah, I agree. Fixed, and thanks. 🙂

**Comment by nhcarrigan** [comment]
As a heads up, you should only need to force push if you've re-written history (i.e. a rebase to resolve conflicts)

**Review by nhcarrigan — APPROVED** [review]

**Comment by nhcarrigan** [comment]
## Thank You

Thank you for your continued contribution to our project. We have reviewed your pull request and are happy to accept these changes.

Please continue to watch for issues labelled `help wanted`, as these will be additional opportunities to contribute.
